grafana/README.md

```diff
@@ -0,0 +1,48 @@
+# Jsonnet Library for Deploying Grafana
+
+This library aims to simplify the deployment of Grafana into Kubernetes
+via Jsonnet.
+
+As well as deploying Grafana itself, it also supports the deployment of
+dashboards, datasources, notification channels and plugins, all from
+within your Jsonnet code.
+
+## Library Notes
+### Statefulness
+Currently, this library creates a stateless Grafana. What this actually
+means is that, on startup, Grafana imports dashboard files into its
+database. However, given Grafana is installed into a deployment without
+an external volume, the database is wiped on restart. This creates a
+stateless Grafana installation.
+
+Statelessness is a **good thing** when deploying all your dashboards, etc
+from Jsonnet code. Through it you avoid your dashboards drifting from the
+Jsonnet/version control managed known good state.
+
+A possible future extension of this library could convert the Grafana
+instance into a stateful one, mounting the database into a PVC.
+
+### Mixins
+This library does not (yet) support Monitoring Mixins directly, although
```

Comment:
Do we have a link to somewhere where we document what a "Monitoring Mixin" is?

Reply:
Yep, I was lazy. Bah :-)